tests/resources/MLTdata-preproduction.ecsv

```diff
@@ -1,209 +1,214 @@
-MLT-201    32.153    14.05    14.05    93.1
-MLT-202    32.137    14.98    14.98    92.1
-MLT-203    32.122    14.76    14.76    93.4
-MLT-204    32.187    14.09    18.33    95.9
-MLT-205    32.142    15.34    15.34    94.9
-MLT-206    32.17     15.48    15.48    94.5
-MLT-207    32.157    15.14    15.14    92.4
-MLT-208    32.179    13.79    13.79    94
-MLT-209    32.199    12.82    12.45    94.8
-MLT-210    32.3      14.72    18.86    93.5
-MLT-211    32.255    10.65    11.39    92.7
-MLT-212    32.269    13.51    15.22    94.6
-MLT-213    32.202    16.47    16.52    95.5
-MLT-214    32.13     15.51    15.51    94.2
-MLT-215    32.047    12.95    15.85    94.9
-MLT-216    32.099    12.36    15.71    94.7
-MLT-218    32.143    14.47    14.47    93.9
-MLT-219    32.096    13.6    13.83    94
-MLT-220    32.045    15.07    17.31    92.1
-MLT-221    32.152    14.56    14.56    94
-MLT-222    31.991    14.75    18.56    94.9
-MLT-223    32.179    12.96    12.96    93
-MLT-224    32.179    12.86    12.86    91.8
-MLT-225    32.21     13.21    13.85    92.5
-MLT-226    32.236    14.29    13.3    94.6
-MLT-227    32.077    13.68    15.27    95.2
-MLT-228    32.001    14.5    17.02    94.2
-MLT-229    32.148    14.22    14.22    91.2
-MLT-230    32.214    15.11    15.2    92.7
-MLT-231    32.11     14.75    14.75    95.4
-MLT-232    32.07     14.5    17.38    94.1
-MLT-233    32.095    12.18    14.19    94.5
-MLT-234    32.086    13.21    15.19    94.5
-MLT-235    32.125    14.98    14.98    95.1
-MLT-236    32.185    16.96    15.34    94.5
-MLT-237    32.085    15.61    18.41    94.4
-MLT-238    32.086    14.82    14.07    95.5
-MLT-239    32.169    12.88    12.88    94.6
-MLT-240    32.097    14.71    15.9     94.6
-MLT-241    32.075    13.88    15.47    92
-MLT-242    32.207    13.17    13.29    91.4
-MLT-243    32.128    13.94    13.94    92.4
-MLT-244    32.18     13.69    13.69    91.7
-MLT-245    32.183    16.72    16.2     92.7
-MLT-246    32.172    13.36    13.36    93.1
-MLT-247    32.233    15.08    14.56    91.4
-MLT-248    32.115    14.87    14.87    93.1
-MLT-249    32.183    14.89    13.57    93.2
-MLT-250    32.163    11.99    11.99    92.9
-MLT-251    32.158    13.64    13.64    92.4
-MLT-252    32.021    14.11    17.84    93.2
-MLT-254    32.25     12.31    12.3     90.2
-MLT-255    32.237    12.71    12.03    93.8
-MLT-256    32.312    14.98    15.78    92.4
-MLT-257    32.101    15.67    15.67    93.6
-MLT-258    32.203    12.74    12.8     93.2
-MLT-259    32.244    12.14    13.41    93.1
-MLT-260    32.284    14.06    13.56    93.3
-MLT-261    32.209    13.91    13.84    93.8
-MLT-262    32.146    15.44    15.44    92.3
+# - {name: mirror_x, unit: cm, datatype: float64}
+# - {name: mirror_y, unit: cm, datatype: float64}
+# - {name: shape_type, datatype: int32}
+# schema: astropy-2.0
+mirror_id mirror_diameter psf_opt psf surface_ref mirror_x mirror_y shape_type
+054 63.614 15.46 20.97 90.2 0.0 0.0 0
```

Comment:
The name of the mirror was changed, this is probably not what we want. I wonder about the change from radius to diameter. It's probably fine, but I would like to alert @GernotMaier to this change.